YAFC/YAFC.csproj

```diff
@@ -2,12 +2,13 @@
   <PropertyGroup>
     <LangVersion>8</LangVersion>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>netcoreapp6.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
this is definitely more correct
however, it's still the incorrect netcoreapp6.0 in the other csproj files, and it should be upgraded to 8.0 anyways

try to keep PRs small and focused and do not put multiple changes into a single PR when possible

Reply:
>  should be upgraded to 8.0 anyways

Someone else already made a ticket for this: #25

Reply:
I would say to leave the original, and we'll upgrade to 8.0 when we work on that ticket. So this one stays clean and only focuses on adding an icon only.

Reply:
Understood